packages/react-vega-lite/package.json

```diff
@@ -18,14 +18,18 @@
     "types"
   ],
   "dependencies": {
-    "react-vega": "^6.0.1"
+    "react-vega": "^6.0.1",
+    "vega-embed": "^4.0.0"
   },
   "peerDependencies": {
     "react": "^16.0.0",
     "vega-lite": "^3.1.0"
```

Comment:
this is redundant because `vega-lite` is already `peerDependencies` of `vega-embed`